src/components/NavBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../assets/css/NavBar.css';
+import { FaMicrophone, FaLessThan } from 'react-icons/fa';
+import { BsFillGearFill } from 'react-icons/bs';
+// import { IoIosArrowBack } from 'react-icons/io';
```

Comment:
- [ ] Please kindly delete the commented line of code to have a clean code base, it's very important to have clean code 🙃 😉

Reply:
Dear @cynthiainga thanks for your feedback. Now everything is okay